mgmt/RecordsConfig.cc

```diff
@@ -1212,6 +1212,8 @@ static const RecordElement RecordsConfig[] =
   //##############################################################################
   {RECT_CONFIG, "proxy.config.ssl.enabled", RECD_INT, "0", RECU_RESTART_TS, RR_NULL, RECC_INT, "[0-1]", RECA_NULL}
   ,
+  {RECT_CONFIG, "proxy.config.ssl.session.ticket.enabled", RECD_INT, "1", RECU_RESTART_TS, RR_NULL, RECC_INT, "[0-1]", RECA_NULL}
```

Comment:
I like proxy.config.ssl.server.ticket_key.enabled.  I think you can enable/disable tickets on client side and server side, so I think it is good to be explicit in case we want to support client side changing advertising ticket capability.